plugin/trino-google-sheets/src/main/java/io/trino/plugin/google/sheets/SheetsConfig.java

```diff
@@ -32,6 +32,8 @@ public class SheetsConfig
     private Optional<String> credentialsFilePath = Optional.empty();
     private Optional<String> credentialsKey = Optional.empty();
     private Optional<String> metadataSheetId = Optional.empty();
+    private String delegatedUserEmail;
```

Comment:
Please move under `credentialsKey` since this config is related to credentials.
Same for getter/setter.